src/env.d.ts

```diff
@@ -4,6 +4,8 @@ declare global {
 			NODE_ENV: "development" | "production";
 			CHECK_GUILD: string;
 			FEEDBACK_CHANNEL: string;
+			ARCHIVE_CHANNEL: string;
```

Comment:
Definitely going to need to add some validation code to check the env is correct.